py3status/modules/wifi.py

```diff
@@ -84,17 +87,27 @@ def __init__(self):
         except:
             pass
 
-    def get_wifi(self):
+    def wifi(self):
         """
         Get WiFi status using iw.
         """
+        if not self.py3.check_commands(['iw']):
+            return {'cache_until': self.py3.CACHE_FOREVER,
+                    'color': self.py3.COLOR_BAD,
+                    'full_text': self.string_unavailable}
```

Comment:
This would be better done in `post_config_hook()` then we are not testing every run.

We are calling the command anyhow so we'd catch the fact it was missing then.